docs/ui/cacao/advanced_cacao_cli.md

```diff
@@ -0,0 +1,98 @@
+
+## Installation
+
+1. The CACAO CLI is a single binary that can be installed on any system. The most recent cli binaries can be downloaded from [the CACAO packages page on Gitlab](https://gitlab.com/cyverse/cacao/-/packages) or you can download from here:
+
+- [CACAO Darwin ARM64](https://gitlab.com/cyverse/cacao/-/package_files/101908036/download)
+- [CACAO Darwin AMD64](https://gitlab.com/cyverse/cacao/-/package_files/101907995/download)
+- [CACAO Windows AMD64](https://gitlab.com/cyverse/cacao/-/package_files/101907940/download)
+- [CACAO Linux ARM64](https://gitlab.com/cyverse/cacao/-/package_files/101907892/download)
+- [CACAO Linux AMD64](https://gitlab.com/cyverse/cacao/-/package_files/101907877/download)
+
+2. If necessary, rename the file to `cacao.zip` and unzip the file. After unzipping, binary will be named `cacao_<os>_<architecture>` e.g. `cacao_darwin_arm64` or `cacao_linux_amd64`.
+
+3. Optional, rename your binary to `cacao` and move into your path, such as `/usr/local/bin`. 
+
+For Linux users, this would be an example of the installation process described above:
+
+```bash
+wget -O cacao.zip  https://gitlab.com/cyverse/cacao/-/package_files/101907877/download
+unzip cacao.zip
+mv cacao_linux_amd64 /usr/local/bin/cacao
+```
+
+## Login with the CACAO CLI
+
+There are currently two approaches to logging in with the CACAO CLI.
+
+### Login Option 1: Login with CIlogin federated identity
+
+This method integrates with CILogin and requires a few steps.
+
+1. `cacao login --browser`
+
+**NOTE: What to do if you encounter a login issue?**
+> Sometimes login using the command line will fail -- a typo happens, a copy-n-paste of a token happens, using the wrong api url happens, etc -- and you need to reset your login. To reset your login, you can use the following command:
+> `cacao logout`
+
+```bash
+Please provide address of Cacao API.
+Format Should be: http://<hostname>:<port>        or    <hostname>:<port> 
+(Developers: this should match the value of API_DOMAIN in install/config.yaml followed by "/api", e.g. ca.cyverse.local/api)
+Cacao API address (http://ca.cyverse.local/api): https://cacao.jetstream-cloud.org/api
+```
+
+2. Enter the CACAO API url: `https://cacao.jetstream-cloud.org/api`
+
+```bash
+Cacao API address (http://ca.cyverse.local/api): https://cacao.jetstream-cloud.org/api
+Please go to this URL in the browser: https://cacao.jetstream-cloud.org/api/user/login
+
+After login, you should get a JSON response, the auth token could be the value of following properties:
+- "IDToken" or "id_token" if keycloak
+- "access_token" if other auth provider
+
+Enter the auth token you get from browser: 
+```
+3. In your browser open the following URL (this is also echoed in the terminal, line #5): https://cacao.jetstream-cloud.org/api/user/login
+
+**WARNING!**
+> For the workshop, use the "ACCESS CI (XSEDE)" identity provider when you login.
```

Comment:
Unclear what "the" workshop is, consider editing if this generalizes beyond a particular workshop.

Reply:
removed the workshop reference and clarified